src/app/components/knob/knob.directive.js

```diff
@@ -24,18 +24,21 @@ class KnobController {
   constructor($scope, $element, $timeout) {
     'ngInject';
     this.timeoutHandler = null;
-    this.$timeout = $timeout;
+    this.timeoutHandler2 = null;
+    this.timeoutHandler3 = null;
+    this.$timeoutService = $timeout;
```

Comment:
expose the services with their name

Reply:
I don't understand, are you questioning my refactor from $timeout to $timeoutService or suggesting do it more consistently on other fields (eg. this.timeoutHandler2 --> this.updateValueTimeoutHandler) ?
- In first case I would like to remind that in naming convection it's important to use multi-words identifiers and that are easy to understand.
- In second case I would like to suggest a whole refactoring for this controller class to make it more readable and  to do it in a separate commit to better have control over regressions and more overall clarity.

Reply:
Angular expose this service $timeout and we are keepint the name of the service when we use it. Feel free to go ahead with the refactor, this directive is quite old

Reply:
Ok, I know in basic AngularJs often happens to use service name by their component name but in this case we are making it an instance field so why not use an understandable name. For the refactor I'll do it if I have time. Thanks.

Reply:
the understandable name is $timeout because Angular expose the service with that name. Is not a Volumio implementation. Renaming the internals library services/filters/etc... is not a good idea (this is a general rule)

Reply:
I have already informed myself and already start using just $timeout for current development, the point is that the $ identifies nicely it as the AngularJs service. 

I never said this was a VolumIO specificity nor to refactor internal services..!